.github/workflows/test-workflow.yml

```diff
@@ -40,11 +40,6 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
-      - name: Install Foundry
```

Comment:
@mshakeg any idea why the Foundry install is still needed in the `test-workflow.yml` for the non foundry executions?
I realize we have to leave it in but curious if we know why it causes the `HardhatFoundryError: Couldn't run `forge`. Please check that your foundry installation is correct` error?

Will restore it but let me know if you have context here.

Reply:
The project now incorporates the [hardhat-foundry plugin](https://hardhat.org/hardhat-runner/plugins/nomicfoundation-hardhat-foundry). This plugin enables Hardhat to utilize dependencies from the `./lib` folder, which are installed using `forge`. Consequently, the plugin tries executing `forge install` to make these dependencies accessible to Hardhat. While the Foundry configuration permits Foundry to utilize dependencies from `./node_modules`, if you think that the contracts won't rely on the `forge`-installed dependencies as is currently the case, you can consider removing the `hardhat-foundry` plugin.

Reply:
I'm sure they will more over time as your PR showed.
Also looking at the OZ library they eventually reached a point of balance with hardhat and foundry support for the same contract so it's not inconceivable in the future we could also.
It's fine as is just wanted to understand the connection since the workflow doesn't run foundry tests explicitly.
Thanks